src/interfaces.ts

```diff
@@ -70,6 +73,16 @@ export interface ReadQueryResult {
   rows: Array<Row>;
 }
 
+export interface ReceiptResult {
```

Comment:
This is mainly for the sake of code readability since the response of an http request cannot be type checked via typescript.